recipes-extended/lsb/lsbinitscripts/functions.patch

```diff
@@ -0,0 +1,33 @@
+From e46b056282c8420f096d5c34d78c00f816788784 Mon Sep 17 00:00:00 2001
+From: Fan Xin <fan.xin@jp.fujitsu.com>
+Date: Mon, 5 Jun 2017 16:26:47 +0900
+Subject: [PATCH 1/2] Upstream-Status: Inappropriate [configuration]
```

Comment:
Could you comment on whether or not this patch is necessary for NILRT; it seems like something we could drop.

Which package ships the `/sbin/consoletype` binary?

Reply:
The patch checks for existence of /sbin/consoletype before using it, so I think it is required - if we want to ship the same version of initscripts.

But we can bump to latest version so this patch wouldn't be required (as the relevant code was [removed](https://github.com/fedora-sysv/initscripts/commit/f88dbd98e992635ee569fb6349f49cb6d215c434)).

None of our feeds contain a package that provides /sbin/console, at-least not the feeds that are shipped with target.

Reply:
Oh. Could you do an evaluation of what it would mean to upgrade the recipe to the latest (or later) version of `sysvinit` then. ie. what commits have gone into the upstream `functions` implementation between SRCREV `a51c1b4f7dcf55b568b2ee4c2b18078849943469` and the new rev?

Reply:
Only change with latest version is that any failures/warnings, etc would now be visible instead of being ignored like in current version. There are no other functional differences.
So I think we can go ahead with upgrading the recipe.